### PR TITLE
Add time range

### LINKED ISSUE #12: Add an option to the functions that allows you to download different timeranges

### PR DESCRIPTION
**Pull request by irisslee**
Fixes #12

### CODE REVIEW

**Comment by palewire** [comment]
@irisslee It looks like the tests are failing, maybe due to hitting NASA a bunch of times in a row. Maybe try adding a `time.sleep` call inside your test loops? 

https://github.com/palewire/nasa-wildfires/runs/4463537897?check_suite_focus=true

File: README.md [hunk 1]
```diff
@@ -71,6 +82,12 @@ Download a regional GeoJSON of hotspots detected by the MODIS satellite in a rec
 >>> data = nasa_wildfires.get_modis('usa-hawaii')
 ```
 
+Download a regional GeoJSON of hotspots detected by the MODIS satellite in a recent 7-day period.
+
+```python
+>>> data = nasa_wildfires.get_modis('usa-hawaii', '7d')
```
Comment (palewire) [inline README.md]: Just to be as explicit as possible, let's use the kwargs here in this example. Like `time='7d'`

File: nasa_wildfires/cli.py [hunk 2]
```diff
@@ -16,32 +16,38 @@ def cmd():
...
 @click.option('-r', '--region', default="Global", type=click.Choice(
     options.REGION_DICT.keys(), case_sensitive=False), help="Hotspot region")
+@click.option('-t', '--time-range', default="24h", type=click.Choice(
+    options.TIME_RANGE, case_sensitive=False), help="Time range")
 @click.option('--indent', default=0, help='Indentation of output')
 @click.option('--sort-keys/--no-sort-keys', default=True, help="Sort the properties keys")
-def modis(region, indent, sort_keys):
-    data = get_modis(region)
+def modis(region, time_range, indent, sort_keys):
+    data = get_modis(region, time_range)
```
Comment (palewire) [inline nasa_wildfires/cli.py:24]: If we call it `time_range` here, we should call it `time_range` in the other file, not `time`. Just for consistency.

**Comment by irisslee** [comment]
The test will now run default and then five random combination of options

**Comment by palewire** [comment]
Thank you!

**Pull request merged**